web/client/plugins/featuregrid/FeatureEditor.jsx

```diff
@@ -292,11 +295,12 @@ const EditorPlugin = compose(
         // due to multiple renders of plugins in contexts (one with default props, then with context props)
         // the options have to be updated when change.
         componentDidUpdate(oldProps) {
-            const newOptions = pick(this.props, ['showFilteredObject', 'showTimeSync', 'timeSync', 'customEditorsOptions']);
-            const oldOptions = pick(oldProps, ['showFilteredObject', 'showTimeSync', 'timeSync', 'customEditorsOptions']);
+            const newOptions = pick(this.props, ['showFilteredObject', 'showTimeSync', 'timeSync', 'customEditorsOptions', 'isSyncWmsActive']);
```

Comment:
i don't think it is this one "isSyncWmsActive" the prop to put here. these should be props that are moved to the state
nd it seems you are not using this info from featuregrid reducer, so please remove this